walk-in-interview/src/main/java/com/google/job/data/JobPayment.java

```diff
@@ -0,0 +1,91 @@
+package com.google.job.data;
+
+/** Class that represents the payment details of a job. */
+public final class JobPayment {
+    private float min;
```

Comment:
Float is not precise enough for money. Maybe use int/long to denote cents instead of dollar?

Reply:
Fixed with int